src/app/grid/grid-summary-formatter/grid-summary-formatter.component.ts

```diff
@@ -0,0 +1,35 @@
+import { DatePipe } from "@angular/common";
+import { Component, ViewChild } from "@angular/core";
+import { IgxDateSummaryOperand, IgxGridComponent, IgxSummaryOperand, IgxSummaryResult } from "igniteui-angular";
+import { DATA } from "../../data/nwindData";
+
+@Component({
+    selector: "app-grid-summary-formatter",
+    styleUrls: ["./grid-summary-formatter.component.scss"],
+    templateUrl: "./grid-summary-formatter.component.html"
+  })
+
+  export class GridSummaryFormatterComponent {
+    @ViewChild("grid1", { read: IgxGridComponent, static: true })
+    public grid1: IgxGridComponent;
+
+    public data: any[];
+
+    public ngOnInit(): void {
+        this.data = DATA;
+    }
+
+    public formatPrice(value: number) {
+        return value < 15 ? 'low' : value > 50 ? 'high' : 'medium';
+    }
```

Comment:
The three samples have some formatter function which are not related to the sample and can be removed.